src/main/resources/META-INF/plugin.xml

```diff
@@ -28,6 +28,7 @@
   <actions>
     <group id="com.redhat.devtools.intellij.tektoncd.tree" popup="true">
       <action class="com.redhat.devtools.intellij.tektoncd.actions.component.OpenEditorTaskAction" text="Open in Editor"/>
+        <action class="com.redhat.devtools.intellij.tektoncd.actions.component.StartTaskDialogAction" text="Start Task"/>
```

Comment:
As the action is on the Task node then text should be only Start (VSCode Tekton uses Start)